pkgs/top-level/ocaml-packages.nix

```diff
@@ -435,6 +435,8 @@ let
 
     functory = callPackage ../development/ocaml-modules/functory { };
 
+    gd4o = callPackage ../development/ocaml-modules/gd4o { };
```

Comment:
I guess this should be called `gd`.

Reply:
in the debian family, Funtoo, and the AUR it seems to be called gd4o so I don't know.

I agree though, the package output implies it's just gd